src/main/java/com/minecolonies/entity/ai/citizen/lumberjack/EntityAIWorkLumberjack.java

```diff
@@ -374,40 +367,40 @@ private boolean plantSapling(BlockPos location)
 
         int saplingSlot = findSaplingSlot();
 
-        if(saplingSlot != -1)
+        if (saplingSlot != -1)
         {
             ItemStack stack = getInventory().getStackInSlot(saplingSlot);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'stack' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)